### PR TITLE
Now facility icons are visible in mobile devices

### LINKED ISSUE #3896: Having images of Facility in mobile devices
**Is your feature request related to a problem? Please describe.**
Currently we can only view the facility image devices greater than 640 px so if we can view it in mobile devices it would be great

**Describe the solution you'd like**
We can place the facility images above the card in mobile devices.

### PR DESCRIPTION
**Pull request by patelaryan7751**
## Proposed Changes
Fixes #3896 
Now facility icons are visible in mobile devices
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by rithviknishad — COMMENTED** [review]
How about using tailwind modifiers like `md:hidden md:visible` instead of useWindowDimensions hook?

**Comment by patelaryan7751** [comment]
@rithviknishad i tried it it didn't work so i just hidden the div when using the hook

**Comment by nihal467** [comment]
@rithviknishad @patelaryan7751 @rithviknishad i can see comments going on, once it is done and finalize for testing, change the label back to need testing

**Comment by rithviknishad** [comment]
@nihal467 you can test.

File: src/Components/Facility/HospitalList.tsx [hunk 1]
```diff
@@ -40,6 +40,7 @@ import * as Notification from "../../Utils/Notifications.js";
 import { Modal } from "@material-ui/core";
 import SelectMenu from "../Common/components/SelectMenu";
 import AccordionV2 from "../Common/components/AccordionV2";
+import useWindowDimensions from "../../Common/hooks/useWindowDimensions";
```
Comment (Ashesh3) [inline src/Components/Facility/HospitalList.tsx]:
```suggestion
```
We do not need to use this, instead simple CSS should work.

File: src/Components/Facility/HospitalList.tsx [hunk 2]
```diff
@@ -81,6 +82,9 @@ export const HospitalList = (props: any) => {
   const userType = currentUser.data.user_type;
   const [notifyMessage, setNotifyMessage] = useState("");
   const [notifyModalFor, setNotifyModalFor] = useState(undefined);
+  const { width } = useWindowDimensions();
+  const smallScreenBreakpoint = 767;
+  const isSmallScreen = width <= smallScreenBreakpoint ? true : false;
```
Comment (Ashesh3) [inline src/Components/Facility/HospitalList.tsx]:
```suggestion
```

File: src/Components/Facility/HospitalList.tsx [hunk 3]
```diff
@@ -343,7 +347,23 @@ export const HospitalList = (props: any) => {
                 )}
               </div>
               <div className="h-full w-full grow">
-                <div className="h-full flex flex-col justify-between w-full">
+                {isSmallScreen ? (
```
Comment (Ashesh3) [inline src/Components/Facility/HospitalList.tsx]:
```suggestion
                <div className="md:hidden">
```

We can use `md:hidden` here, to hide this div for screen sizes >= 768px. Achieving the same result.

File: src/Components/Facility/HospitalList.tsx [hunk 4]
```diff
@@ -343,7 +347,23 @@ export const HospitalList = (props: any) => {
...
+                        alt={facility.name}
+                        className="w-full h-full object-cover"
+                      />
+                    )) || (
+                      <i className="fas fa-hospital text-4xl block text-gray-500 p-10" />
+                    )}
+                  </div>
+                ) : (
+                  " "
+                )}
```
Comment (Ashesh3) [inline src/Components/Facility/HospitalList.tsx]:
```suggestion
                </div>
```

Comment (Ashesh3) [reply src/Components/Facility/HospitalList.tsx]: The final result:

image

**Review by Ashesh3 — CHANGES_REQUESTED** [review]

**Comment by Ashesh3** [comment]
> How about using tailwind modifiers like `md:hidden md:visible` instead of useWindowDimensions hook?

I think this should be the proper approach, I tried to use the above modifiers and was able to achieve the same results.

**Comment by nihal467** [comment]
@patelaryan7751 @Ashesh3 i can see the comments going on, once its finalise, mark it for need testing

**Comment by patelaryan7751** [comment]
Hey @Ashesh3 i have done the above said changes.

**Review by Ashesh3 — APPROVED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**